### PR TITLE
Update Tinkerbell stack images to latest release versions

### PR DESCRIPTION
**Pull request by chrisdoherty4**
Update the Tinkerbell stack images to the latest released versions.

|App|Version|
|---|---|
|Rufio|v0.2.1|
|Tink|v0.8.0|
|Hegel|v0.10.1|
|CAPT|v0.4.0|

These changes introduce fixes that enable removal of disk constraints in the Bare Metal EKS-A provider. The chart that consumes the images will be updated separately.

### CODE REVIEW

File: projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@
 # limitations under the License.
 
 # This file was generated by running `make generate`. DO NOT EDIT
+
```
Comment (abhay-krishna) [inline projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml:16]: What caused this newline addition in the generated files?

Comment (chrisdoherty4) [reply projects/aws/bottlerocket-bootstrap/buildspecs/batch-build.yml:16]: I don't know, its generated. This happened when I ran `make generate` from the root of the project

File: projects/tinkerbell/rufio/patches/0001-Patch-BMC-response-interrogation.patch [hunk 2]
```diff
@@ -0,0 +1,30 @@
+From c6aea9e215c88cab84f2235adb22fe6249b17b38 Mon Sep 17 00:00:00 2001
```
Comment (pokearu) [inline projects/tinkerbell/rufio/patches/0001-Patch-BMC-response-interrogation.patch:1]: I think we had fixed this in the latest rufio version? 
The old patch would not be needed as well.

Comment (chrisdoherty4) [reply projects/tinkerbell/rufio/patches/0001-Patch-BMC-response-interrogation.patch:1]: I think you're right. I released v0.2.0, not sure why I used v0.1.0 (which doesn't have the fix).

**Comment by chrisdoherty4** [comment]
These changes need to be synchronized with changes to the EKS-A repository so holding until those are in-place.

**Comment by jacobweinstock** [comment]
Looks like there arent any references to Boots in the files changed section so I believe only the initial PR comment should updated in terms of the Boots version. `v0.8.0` -> `v0.8.1`

**Comment by chrisdoherty4** [comment]
> Looks like there arent any references to Boots in the files changed section so I believe only the initial PR comment should be updated in terms of the Boots version. `v0.8.0` -> `v0.8.1`

Thanks, fixed a few errors in the PR overview.

File: projects/tinkerbell/hegel/Makefile [hunk 3]
```diff
@@ -10,23 +10,12 @@ BASE_IMAGE_NAME?=eks-distro-minimal-base
...
 
-$(GATHER_LICENSES_TARGETS): $(FIX_LICENSES_STRPTIME_TARGET)
-
-$(FIX_LICENSES_STRPTIME_TARGET): | $(GO_MOD_DOWNLOAD_TARGETS)
-# One of the project dependencies github.com/pbnjay/strptime does not have a license file.
-# The project owner had mentioned a MIT License and has it embedded in github.com/pbnjay/strptime/strptime.go.
-# We copied the license out to STRPTIME_LICENSE.txt and moved it into vendor during build.
-	cp STRPTIME_LICENSE.txt $(REPO)/vendor/github.com/pbnjay/strptime/LICENSE.txt
-
-
```
Comment (abhinavmpandey08) [inline projects/tinkerbell/hegel/Makefile:29]: why is this being removed from the makefile?

File: projects/tinkerbell/rufio/README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ## **Rufio**
-![Version](https://img.shields.io/badge/version-a4053e5c1e7f32fb5c0a9962846c219c3ef8aaf3-blue)
+![Version](https://img.shields.io/badge/version-v0.1.0-blue)
```
Comment (abhinavmpandey08) [inline projects/tinkerbell/rufio/README.md:2]: should this be `v0.2.1`
```suggestion
![Version](https://img.shields.io/badge/version-v0.2.1-blue)
```

File: projects/tinkerbell/cluster-api-provider-tinkerbell/README.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ## **Cluster API Provider for Tinkerbell**
-![Version](https://img.shields.io/badge/version-9e9c2a397288908f73a4f499ac00aaf96d15deb6-blue)
+![Version](https://img.shields.io/badge/version-v0.5.0-blue)
```
Comment (abhinavmpandey08) [inline projects/tinkerbell/cluster-api-provider-tinkerbell/README.md:2]: this should be `v0.4.0` based on the git tag under capt
```suggestion
![Version](https://img.shields.io/badge/version-v0.4.0-blue)
```

File: projects/tinkerbell/hegel/Makefile (hunk 3 above)
Comment (chrisdoherty4) [reply projects/tinkerbell/hegel/Makefile:29]: `strptime` is no longer a dependency of Hegel.

File: projects/tinkerbell/rufio/README.md (hunk 4 above)
Comment (abhinavmpandey08) [reply projects/tinkerbell/rufio/README.md:2]: looks like this fix got reverted

**Review by abhinavmpandey08 — APPROVED** [review]

**Review by jacobweinstock — APPROVED** [review]

**Pull request merged**